homeassistant/components/seventeentrack/sensor.py

```diff
@@ -111,242 +95,150 @@ async def async_setup_entry(
 ) -> None:
     """Set up a 17Track sensor entry."""
 
-    client = hass.data[DOMAIN][config_entry.entry_id]
+    coordinator: SeventeenTrackCoordinator = hass.data[DOMAIN][config_entry.entry_id]
+
+    @callback
+    def _async_create_remove_entities():
+        remove_packages(hass, coordinator.account_id, coordinator.data.old_packages)
+
+        async_add_entities(
+            SeventeenTrackPackageSensor(
+                coordinator, t_number, p_data.friendly_name, p_data.status
+            )
+            for t_number, p_data in coordinator.data.new_packages.items()
```

Comment:
I'd write out the variables, we write code for developers, not for the computer :)